WordPress/Sniffs/Theme/DiscouragedIframeSniff.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Check for use of iframe. Often used for malicious code.
+ *
+ * @link    https://make.wordpress.org/themes/handbook/review/required/#core-functionality-and-features
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.xx.0
+ */
+class WordPress_Sniffs_Theme_DiscouragedIframeSniff implements PHP_CodeSniffer_Sniff {
+
+	/**
+	 * The regex to catch the blacklisted attributes.
+	 *
+	 * @var string
+	 */
+	protected $iframe_regex;
+
+	/**
+	 * Returns an array of tokens this test wants to listen for.
+	 *
+	 * @return array
+	 */
+	public function register() {
+		$this->iframe_regex = '/<(iframe)[^>]*>/';
```

Comment:
@grappler If by "fail", you mean: would result in an error being thrown: true, but the same goes for the current regex, which is why I suspect the regex will still change ;-)